go.mod

```diff
@@ -1,24 +1,24 @@
 module github.com/minio/hperf
 
-go 1.22
+go 1.22.4
```

Comment:
```suggestion
go 1.22
```

Whatever script you have that does this automatically, kill it.

Reply:
it's just go mod init and tidy